lib/simulator-xcode-9.js

```diff
@@ -498,6 +498,7 @@ class SimulatorXcode9 extends SimulatorXcode8 {
   /**
    * @typedef {Object} LanguageOptions
    * @property {string} name The name of the language, for example `de` or `zh-Hant-CN`
+   * @property {boolean} skipSyncUiDialogTranslation skip sync UI dialog translation when true
```

Comment:
`@property {boolean} [skipSyncUiDialogTranslation] no Simulator services will be reset if this option is set to true. See https://github.com/appium/appium/issues/19440 for more details`

Reply:
Thanks! Updated as https://github.com/appium/appium-ios-simulator/pull/404/commits/dd6d69fc69e267360519d8f9430906e842cb3721